Assets/Scripts/Manager/StorySceneManager.cs

```diff
@@ -575,6 +582,20 @@ IEnumerator move(Transform objectToMove, Transform target, float speed)
 		doorToOpen.GetComponent<Animator> ().SetBool ("open", true);
 		doorToOpen.GetComponent<BoxCollider2D> ().enabled = false;
 	}
-		
-		
+
+    //Enable the black panel that is used for both endings
+    private void enableBlackPanel(Color textColor) {
+        blackPanel.SetActive(true);
+        blackPanelText.gameObject.SetActive(true);
+        blackPanelText.color = textColor;
+        textBoxManager.theText = blackPanelText;
+    }
+
+    private void theEnd(GameObject panel, bool fadeOutPanel) {
+        Animator theEndAnimator = panel.GetComponent<Animator>();
+        theEndAnimator.SetBool("fadeOutPanel", fadeOutPanel);
+        theEndAnimator.SetBool("startEnd", true);
+        Debug.Log("The end fade start!");
+        //TODO: NEED TO WRITE IENUMERATOR THAT WAITS FOR ANIMATION TO END THEN SWITCHES SCENES
```

Comment:
Yes this is just a TODO to transition to the end credits scene. I will fix this part when I do the end credits.